neuron/src-bash/neuron-search

```diff
@@ -9,6 +9,8 @@ EXTENSIONS=${4}
 OPENCMD=$(envsubst -no-unset -no-empty <<<${5})
 EXACT=""
 
+: ${NEURON_SEARCH_CONTEXT_ABOVE:=5}
```

Comment:
Where can this be documented? @srid 

Something like

> Changed the context in search view by setting the environment variable `NEURON_SEARCH_CONTEXT_ABOVE`, ex:
>
> ```sh
> # inside my ~/.bashrc
>
> # now search term will be on first line in preview
> export NEURON_SEARCH_CONTEXT_ABOVE=0
> ```

Reply:
Why should this parameter to be made configurable in the first place? Does this affect the behaviour of (non-full-text) `neuron search`?

Reply:
This only affects the visual behaviour. And yes, it affects both fulltext and exact searching